test/manual/authentication/pytest_crypto.py

```diff
@@ -11,7 +11,9 @@
 def test_hashing():
     with open("/vagrant/hiera/settings.yaml", 'r') as stream:
         try:
-            cryptopath = yaml.load(stream)['crypto']['path']
+            yamlres = yaml.load(stream)
+            root = yamlres['general']['root']
+            cryptopath = root + yamlres['crypto']['path']
         except yaml.YAMLError as exc:
             print exc
```

Comment:
Sometimes (most often) sysadmins, or developers will have the application running as a background process.  So, `print exc` will be printed, and forgotten in the background.  Instead, you should implement the [`Logger`](https://github.com/jeff1evesque/machine-learning/blob/9a5dc33891f06a56cebabf6872af866eeb1a4ab9/brain/database/db_query.py#L39) class, so the exception gets logged into a corresponding log file.  However, I'd classify that as an application error, so I'd implement the following in `pytest_crypto.py`:

```python
...
self.logger = Logger(__name__, 'error', 'database', 'crypto')
...
```

**Note:** you'll find more information in the docblock of [`logger.py`](https://github.com/jeff1evesque/machine-learning/blob/9a5dc33891f06a56cebabf6872af866eeb1a4ab9/log/logger.py#L26-L57).  Also, could you change `exc` to `error` (just trying to be consistent with the codebase).